### DISCUSSION TITLE
Widget Scroll

### DISCUSSION THREAD

**Discussion by PaoloPetra** [iTwin/itwinjs-core#3833] [Q&A] [answered]
Hello everyone
I have inserted the charts in a widget, since there are more than one, the widget allows me to scroll vertically through these charts
my problem arises when I have to show the tooltips of these charts
as I cannot determine how many vertical scrolls I have made to have a y value such as to be able to view the tooltips.
So re-assemble
is there a property in the widget that allows me to determine the scrolls I've made?

thanks for your invaluable help
Paul

**Comment by bsteinbk** [comment, marked as answer]
There is nothing in the Widget Layout system that would help out here. It is up to each individual widget to maintain knowledge about its own internal state.
